application/controllers/handler.js

```diff
@@ -82,12 +83,12 @@ module.exports = {
       email: decodeURI(req.payload.email).toLowerCase().replace(/\s/g,''),
       password: decodeURI(req.payload.password)
     };
-    console.log('query: ', query);
+    console.log('search query: ', query.email, query.password.substr(0, 5)+'...');
```

Comment:
its hard to log something here in order to help a developer debug when something happend in this route. But passwords are secret thus I will just remove the logging of it.